### PR TITLE
issues-1: global styles

### PR DESCRIPTION
**Pull request by QuentinVecchio**
This PR introduces the following changes:
- Add fonts in lazy styles
- Add global variables for fonts, color, background

Part of #1 

Test URLs:
- Before: https://main--durysta--hlxsites.hlx.page
- After: https://1-global-styles--durysta--hlxsites.hlx.page

### CODE REVIEW

File: styles/lazy-styles.css [hunk 1]
```diff
@@ -1 +1,30 @@
-/* below the fold CSS goes here */
\ No newline at end of file
+/* below the fold CSS goes here */
+@font-face{
+    font-family:"ITC Avant Garde Gothic W01 Bk";
```
Comment (kmurugulla) [inline styles/lazy-styles.css:3]: Could you please explore the option of downloading the font and check in into the source code, refer to it from source code which will avoid calls to fast.fonts.net ?

File: styles/styles.css [hunk 2]
```diff
@@ -15,12 +15,13 @@
   --link-color: #035fe6;
```
Comment (kmurugulla) [inline styles/styles.css:15]: hlx import , eye dropper tool provided the following which could be used as starting point. please ignore if you have already obtained them using eye dropper tool.

/* fonts */
--heading-font-family: sans-serif;
--body-font-family: "ITC Avant Garde Gothic W01 Md";
--fixed-font-family: "Roboto Mono", menlo, consolas, "Liberation Mono", monospace;

/* colors */
--text-color: #212529;
--link-color: #007bff;
--link-hover-color: #007bff;
--background-color: #ffffff;
--overlay-background-color: #eeeeee;
--highlight-background-color: #cccccc;

/* sizes */
--heading-font-size-xxl: 40px;
--heading-font-size-xl: 32px;
--heading-font-size-l: 28px;
--heading-font-size-m: 24px;
--heading-font-size-s: 20px;
--heading-font-size-xs: 16px;
--body-font-size-l: 23px;
--body-font-size-m: 18px;
--body-font-size-s: 16px;
--body-font-size-xs: 14px;
--body-font-size-xxs: 12.992px;
--body-font-size-undefined: 12px;

File: styles/lazy-styles.css (hunk 1 above)
Comment (QuentinVecchio) [reply styles/lazy-styles.css]: I downloaded the fonts and put them under `styles`. The fonts are now loaded from source

File: styles/styles.css [hunk 3]
```diff
@@ -15,12 +15,13 @@
...
-  --overlay-background-color: #eee;
+  --overlay-background-color: #00806e;
   --highlight-background-color: #ccc;
-  --text-color: #000;
+  --overlay-color: #fff;
+  --text-color: #414042;
 
   /* fonts */
-  --body-font-family: 'helvetica neue', helvetica, ubuntu, roboto, noto, sans-serif;
+  --body-font-family: 'ITC Avant Garde Gothic W01 Md';
```
[the remark this answers is not in the document]
Comment (QuentinVecchio) [reply styles/styles.css]: I used as fallback fonts the ones defined in `--fixed-font-family`. Let me know if that makes sense

**Pull request merged**